references/video_classification/train.py

```diff
@@ -149,11 +150,14 @@ def main(args):
     print("Loading validation data")
     cache_path = _get_cache_path(valdir)
 
-    if not args.weights:
-        transform_test = presets.VideoClassificationPresetEval((128, 171), (112, 112))
+    if not args.prototype:
+        transform_test = presets.VideoClassificationPresetEval(resize_size=(128, 171), crop_size=(112, 112))
     else:
-        weights = PM.get_weight(args.weights)
-        transform_test = weights.transforms()
+        if args.weights:
+            weights = PM.get_weight(args.weights)
+            transform_test = weights.transforms()
+        else:
+            transform_test = PT.Kinect400Eval(crop_size=(112, 112), resize_size=(128, 171))
```

Comment:
I confirm that the arguments in the two classes take the parameters in different order. We are going to remove the old presets anyway and stick with the second one.